api/v1/tests/test_helper.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+
+from app.auth.helper import (email_validator,
+                             address_validator,
+                             password_validator,
+                             user_name_validator)
+
+
+class TestHelper(unittest.TestCase):
+
+    def test_valid_email(self):
+        self.assertTrue(email_validator('quantum@gmail.com'))
```

Comment:
Suggested missing edge case:
- Test whether an invalid email is valid according to your validator.

Reply:
Do so for the other validators too.

Reply:
Thanks for your feedback.I am implementing it